python/lancedb/table.py

```diff
@@ -188,27 +188,35 @@ def create_index(
         num_partitions=256,
         num_sub_vectors=96,
         vector_column_name=VECTOR_COLUMN_NAME,
+        replace: bool = True,
     ):
         """Create an index on the table.
 
         Parameters
         ----------
         metric: str, default "L2"
-            The distance metric to use when creating the index. Valid values are "L2" or "cosine".
+            The distance metric to use when creating the index.
+            Valid values are "L2", "cosine", or "dot".
             L2 is euclidean distance.
         num_partitions: int
             The number of IVF partitions to use when creating the index.
             Default is 256.
         num_sub_vectors: int
             The number of PQ sub-vectors to use when creating the index.
             Default is 96.
+        vector_column_name: str
+            The vector column name to create the index.
```

Comment:
nit: add "defaults to "vector"" in the doc

Reply:
Done